### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by georgeijidola**

### CODE REVIEW

File: src/docs/ws.json [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+  "servers": [
+    {
+      "url": "http://localhost:{port}/chat/{chatId}/send",
+      "description": "Allows you to connect using Websocket.",
+      "schema": "ws",
+      "variables": {
+        "chatId": {
+          "default": "397fed73-4bcb-49ea-938b-74d4fe5ce811",
+          "description": "You can get your `chatId` from the get chats REST endpoint"
```
Comment (marshallswain) [inline src/docs/ws.json]: from the `GET /chats`

File: src/docs/ws.json [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    },
+
+    {
+      "url": "https://dev.angular-ngrx-chat.com/chat/{chatId}/send",
+      "description": "Allows you to connect using Websocket.",
+      "schema": "wss",
+      "variables": {
+        "chatId": {
+          "default": "397fed73-4bcb-49ea-938b-74d4fe5ce811",
+          "description": "You can get your `chatId` from the get chats REST endpoint"
```
Comment (marshallswain) [inline src/docs/ws.json]: from the `GET /chats`

File: src/docs/ws.json [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        },
+        "port": {
+          "default": "8080"
+        }
+      }
+    },
+
+    {
+      "url": "https://dev.angular-ngrx-chat.com/chat/{chatId}/send",
```
Comment (marshallswain) [inline src/docs/ws.json:26]: Is this a repeat of the previous entry?

File: src/docs/ws.json [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            "userId": {
+              "type": "string",
+              "format": "uuid",
+              "description": "The user's unique identifier",
+              "example": "5d99e226-7891-4ec6-8016-59ac86e1ace7"
+            },
+            "username": {
+              "type": "string",
+              "description": "Username of the user",
+              "example": "Marshall"
```
Comment (marshallswain) [inline src/docs/ws.json:110]: That's me! 🎉   :)

File: src/docs/ws.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+> Example payload
+
+```json
+
+"Hello world!"
+```
+
+*Action to send a message.*
+
+Action to send a message.
```
Comment (marshallswain) [inline src/docs/ws.md]: Is this meant to be here a second time?

File: src/docs/ws.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+
+#### Headers
+
+##### Properties
+
+|Name|Type|Required|Description|
```
Comment (marshallswain) [inline src/docs/ws.md:65]: I suggest getting rid of the tables.  Totally optional, but it's easier to maintain plain text docs than tables.  Here's an example: https://v1.feathers-pinia.pages.dev/guide/service-stores.html#using-definestore.

I use the following format:

```md
- `attrName {Type}` a brief and complete description. default defaultValue.  *required*
```

- `attrName {Type}` a brief and complete description. default defaultValue  **required**

Comment (marshallswain) [reply src/docs/ws.md:65]: You're free to leave it as a table if you'd like.  I've adopted this preference after maintaining open source docs for a while.

File: src/routes/index.js [hunk 7]
```diff
@@ -8,10 +8,122 @@ const Password = require('../helpers/password')
...
+      const markdownOutput = await readFile(
+        path.resolve(__dirname, '../docs/ws.md'),
+        'utf8'
+      )
+
+      reply.type('text/html').send(md.render(markdownOutput))
+    })
+  )
+
+  // fastify.route({
```
Comment (marshallswain) [inline src/routes/index.js]: What's the purpose of the giant commented section?   If it has future relevance then leave it.

**Review by marshallswain — APPROVED** [review]
Looks good.  A couple of small suggestions and questions.  then :shipit:

File: src/docs/ws.json (hunk 3 above)
Comment (georgeijidola) [reply src/docs/ws.json:26]: sort of but the first is the local server and this is dev

File: src/docs/ws.md (hunk 6 above)
Comment (georgeijidola) [reply src/docs/ws.md:65]: this is actually a generated document by widdershins

Comment (georgeijidola) [reply src/docs/ws.md:65]: I am not sure there is nothing I can do about the table

**Review by marshallswain — APPROVED** [review]

**Pull request merged**